react/src/js/react/posts/CreatePost.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import Reflux from "reflux";
+
+import { FormGroup, FormControl, Form, Button, Row, Col } from "react-bootstrap";
+import PrivacyDropdown from "./PrivacyDropdown";
+
+import "../../../scss/CreatePost.scss";
```

Comment:
if you import your css files into main.scss then they will be imported to the whole project and you do not have to import them here.